exporting/tests/test_exporting_engine.c

```diff
@@ -1194,9 +1194,10 @@ static void test_prometheus_remote_write_send_header(void **state)
         "POST /receive HTTP/1.1\r\n"
         "Host: test-host\r\n"
         "Accept: */*\r\n"
+        "X-Prometheus-Remote-Write-Version: 0.1.0\r\n"
         "Content-Length: 11\r\n"
         "Content-Type: application/x-www-form-urlencoded\r\n\r\n");
-    expect_value(__wrap_send, len, 125);
+    expect_value(__wrap_send, len, 167);
```

Comment:
Only to confirm, both the current string and the previous string do not have the correct length, the previous had a difference of 13 bytes and now we have difference of 15 values according `kate`:

![image6](https://user-images.githubusercontent.com/49162938/84378471-bfa60880-abd3-11ea-94e0-fa63caefe7c7.png)
 
Is the expected?

Reply:
It's because you are counting in escape characters as well.